README.md

```diff
@@ -54,15 +54,19 @@ pushd kythe-verification; stack install && ./test.sh; popd
 
 # Demo
 
-To index a few packages and serve the index, run:
+To index a few packages, run:
 
-```
+```bash
 ./build-stack.sh /tmp/logs mtlparse cpu
 ```
 
-The script temporarily replaces the system GHC with
-`wrappers/stack-docker/fake-stack/ghc` script, does the indexing and serves the
-built index at `localhost:8080`.
+The script installs a post hook to `stack path --compiler-exe`, does the indexing when `ghc --make` is detected to build a package. You can run `build-stack.sh` multiple times.
```

Comment:
I used '--static' ad-hoc since it seemed to be only called once. Could you check if '--make' is only called once (so we avoid reindexing)?